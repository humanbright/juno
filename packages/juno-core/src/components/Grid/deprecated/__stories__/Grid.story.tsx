import React from 'react';

import { RcGrid, RcPaper } from '@ringcentral/juno';

export default {
  title: 'ðŸ–¤ Deprecated Components/Grid',
};

export const Grid = () => (
  <div>
    <RcGrid container spacing={9}>
      <RcGrid item xs={12}>
        <RcPaper>xs=12</RcPaper>
      </RcGrid>
      <RcGrid item xs={6}>
        <RcPaper>xs=6</RcPaper>
      </RcGrid>
      <RcGrid item xs={6}>
        <RcPaper>xs=6</RcPaper>
      </RcGrid>
      <RcGrid item xs={3}>
        <RcPaper>xs=3</RcPaper>
      </RcGrid>
      <RcGrid item xs={3}>
        <RcPaper>xs=3</RcPaper>
      </RcGrid>
      <RcGrid item xs={3}>
        <RcPaper>xs=3</RcPaper>
      </RcGrid>
      <RcGrid item xs={3}>
        <RcPaper>xs=3</RcPaper>
      </RcGrid>
    </RcGrid>
  </div>
);
