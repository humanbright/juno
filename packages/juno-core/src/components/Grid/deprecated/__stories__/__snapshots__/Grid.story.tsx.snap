// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ðŸ–¤ Deprecated Components/Grid Grid 1`] = `
<div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-9"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=12
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=6
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=6
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=3
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=3
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=3
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
      >
        xs=3
      </div>
    </div>
  </div>
</div>
`;
